python-sdk/src/langwatch/telemetry/tracing.py

```
@@ -332,6 +332,7 @@ def evaluate(
         conversation: Conversation = [],
         settings: Optional[Dict[str, Any]] = None,
         as_guardrail: bool = False,
+        data: Optional[Union[BasicEvaluateData, Dict[str, Any]]] = None,
```

Ruff (0.8.2)

Undefined name `BasicEvaluateData`

(F821)